src/styles/calendar.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: $gap;
+  padding: $padding;
+
+  &__cell {
+    @include square($cellWidth);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #000;
+    background-color: #eee;
+    transition: transform 0.5s, background-color 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__cell--#{$index}::before {
+      content: "#{$index}";
```

Comment:
you can write &__cell:nth-child(#{$index})

Reply:
Agree, you should use nth-child, also remove number froms class in html